README.md

```diff
@@ -1,6 +1,64 @@
-# edgex-endorsement
+# EdgeX Endorsement
 
 ## Introduction
 
-Code, tools, and documentation for validating and endorsing device profiles written by third parties.
+Code, tools, and documentation for validating and endorsing device profiles written by third parties. At a high level you'll need to complete five steps to be eligible for the EdgeX Endorsement Program which will ensure that data provided meets all required components. 
+
+1) Deploy EdgeX using one of the provided cloud templates. Alternatively, you can leverage the latest released compose-file provided here: This ensures a vanilla version of EdgeX is being used without modification to ensure the highest compatibility.
+2)  Compose and deploy your device profile using the Device Profile Modeling tool provided [here](https://www.edgexfoundry.org/tbd). This will ensure the profile is validated and has all required components.
+2) Run your device locally, configured to point at the cloud instance of the deployed EdgeX to send data to either an MQTT or REST device service.
+3) Once deployed, Run the verification script provided to ensure that core-data has been correctly populated with the data that was sent in the previous step.
+4) Submit your device profile along with sample data to EdgeX for approval [here](https://www.edgexfoundry.org/tbd).
+
+
+
+## 1) Deploy EdgeX
+
+If you want to see how all of EdgeX works - you can leverage your own Azure account and deploy EdgeX to the cloud. This template leverages Azure Container Instances and will deploy a single group called "edgex-example" with 12 services deployed with 2.4 vCPUs allocated (0.2 vCPUs per service) and 6GB of RAM allocated (0.5 GB per service) with an estimated cost of $0.14904 / hour or $3.57696 / day. 
```

Comment:
If people want to this to run in their own environment vs. Azure - will we allow it?  If so, do we send them to our regular docs?

Reply:
My original thought was no...avoid any issue with changes in configuration, customizations of EdgeX, or other potentially not foreseeable changes that could be done locally. But, maybe it doesn't matter if we have verification scripts.